lib/url.js

```diff
@@ -585,8 +585,8 @@ Url.prototype.format = function format() {
     host = auth + this.host;
   } else if (this.hostname) {
     host = auth + (this.hostname.indexOf(':') === -1 ?
-        this.hostname :
-        '[' + this.hostname + ']');
+      this.hostname :
```

Comment:
This one is tricky to fix that way because the custom `align-multiline-assignment` rule seems to conflict with the newer indent stuff. I'd been thinking it might be good to remove that custom rule anyway, but I don't want to do it ahead of time.

There are some other ways to make this code clearer, but they will require benchmarks to confirm that they don't adversely affect performance. Stay tuned.